src/renderer.ts

```diff
@@ -1,6 +1,6 @@
 import * as VF from 'vexflow';
 import { Producer } from './producer';
-import { ClefMessage, EasyScoreMessage, NoteMessage } from './types';
+import { EasyScoreMessage, MeasureStartMessage, NoteMessage } from './types';
 
 export class Renderer {
   static render(elementId: string, musicXml: string): void {
```

Comment:
When I make the width and height of the `VF.Factory` renderer large, I can see the entire score:

![image](https://user-images.githubusercontent.com/19232300/161957964-acf5a1b5-e191-494a-a57e-d73e84cc4a82.png)

Is there a way to auto-size the renderer?

Reply:
I will take that on the next PR